.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+### Issue Reference
+This PR addresses the Issue : Fixes #
```

Comment:
Might be a good idea to add a note here to check Github issues for any dupes. Eg:

Note: Please check the Issues on this repo first to make sure you are not duplicating work. Go here: https://github.com/IEEEKeralaSection/rescuekerala/issues